lte/gateway/c/session_manager/test/test_set_session_manager_handler.cpp

```diff
@@ -192,12 +192,24 @@ TEST_F(SessionManagerHandlerTest, test_SetAmfSessionContext) {
       &server_context, &request,
       [this](grpc::Status status, SmContextVoid Void) {});
 
+  assert_sm_session_context(&request);
+  EXPECT_EQ(request.mutable_common_context()->sid().id(), IMSI1);
+  auto req =
+      request.mutable_rat_specific_context()->mutable_m5gsm_session_context();
+
   // Run session creation in the EventBase loop
   evb->loopOnce();
+  SessionConfig cfg;
+  cfg.common_context       = request.common_context();
+  cfg.rat_specific_context = request.rat_specific_context();
+  cfg.rat_specific_context.mutable_m5gsm_session_context()->set_ssc_mode(
+      SSC_MODE_3);
 
   auto session_map = session_store->read_sessions({IMSI1});
   auto it          = session_map.find(IMSI1);
   EXPECT_FALSE(it == session_map.end());
+  auto pdu_id = req->pdu_session_id();
+  set_session_manager->send_create_session(session_map, IMSI1, cfg, pdu_id);
```

Comment:
In my opinion, this testcase can do much more: why aren't we checking every single parameter of the session just created.

Note to be effective, unit test must check the result values to see if what you created is what you expect. 

This applies for all the tests in this PR or session manager.